NEWS.md

```diff
@@ -1,5 +1,8 @@
 # pkgdown (development version)
 
+* pkgdown now supports local searching so no set-up is needed for users to
```

Comment:
```suggestion
* pkgdown now supports local searching. It is enabled by default because no set-up is needed for users to
```

Also need a suggestion as to what to do if you previously used algolia